pennylane/fermi/fermionic.py

```diff
@@ -511,6 +512,71 @@ def from_string(fermi_string):
     return FermiWord({(i, int(s[:-1])): s[-1] for i, s in enumerate(operators)})
 
 
+def _to_string(fermi_op, of=False):
+    r"""Return a string representation of the :class:`~.FermiWord` or :class:`~.FermiSentence` object.
```

Comment:
Please remove the support for `FermiSentence` unless you need it in other functions. That simplifies the function and there will be no need for dispatching.